### PR TITLE
Miscellaneous

### PR DESCRIPTION
**Pull request by psiipher**

### CODE REVIEW

File: Example/PrebidDemoKotlin/src/main/java/org/prebid/mobile/prebidkotlindemo/activities/ads/gam/original/GamOriginalApiDisplayBanner320x50Activity.kt [hunk 1]
```diff
@@ -16,61 +16,91 @@
...
+
+        val request = AdManagerAdRequest.Builder().build()
+        adUnit?.fetchDemandAndLoad(adView, request, object: GamEventListener {
             override fun onAdLoaded() {
-                super.onAdLoaded()
-                AdViewUtils.findPrebidCreativeSize(adView, object : AdViewUtils.PbFindSizeListener {
-                    override fun success(width: Int, height: Int) {
-                        adView.setAdSizes(AdSize(width, height))
-                    }
+                Log.d("Tushar", "onAdLoaded")
```
Comment (nikhilAgarwalM) [inline Example/PrebidDemoKotlin/src/main/java/org/prebid/mobile/prebidkotlindemo/activities/ads/gam/original/GamOriginalApiDisplayBanner320x50Activity.kt:63]: remove tushar from logs

File: network/src/main/java/com/app/network/builder/RetrofitBuilder.kt [hunk 2]
```diff
@@ -18,8 +18,9 @@ class RetrofitBuilder<T>(
         Moshi.Builder().build()
     private val okHttpClient by lazy { OkHttpClient.Builder() }
     private val retrofit by lazy {
+        val correctBaseUrl = if (baseUrl.endsWith("/")) baseUrl else baseUrl.plus("/")
         val builder = Retrofit.Builder()
-            .baseUrl(baseUrl)
+            .baseUrl(correctBaseUrl)
```
Comment (nikhilAgarwalM) [inline network/src/main/java/com/app/network/builder/RetrofitBuilder.kt:23]: correctBaseUrl?  we need to rethink on the name

File: Example/PrebidDemoKotlin/src/main/java/org/prebid/mobile/prebidkotlindemo/activities/ads/gam/original/GamOriginalApiDisplayBanner320x50Activity.kt (hunk 1 above)
Comment (psiipher) [reply Example/PrebidDemoKotlin/src/main/java/org/prebid/mobile/prebidkotlindemo/activities/ads/gam/original/GamOriginalApiDisplayBanner320x50Activity.kt]: Done.

File: network/src/main/java/com/app/network/builder/RetrofitBuilder.kt (hunk 2 above)
Comment (psiipher) [reply network/src/main/java/com/app/network/builder/RetrofitBuilder.kt]: 😂😂
Done

**Review by nikhilAgarwalM — APPROVED** [review]

**Pull request merged**